stubs/tensorflow/tensorflow/config/__init__.pyi

```diff
@@ -0,0 +1,11 @@
+from typing import NamedTuple
```

Comment:
For files that are incomplete stubs you should always add,

```python
def __getattr__(name: str) -> Incomplete: ...
```

The stubs I shared do not do this since codebase I work on I tend to prefer untyped methods being false positive vs false negative. But for typeshed/general users we don't want incomplete stubs to create false positives. So until module stubs has every publicly documented member it should always have  `__getattr__` Incomplete. Same comment for the other file.

Reply:
Fixed in 13afbe415